src/parser/__main__.py

```diff
@@ -70,7 +70,8 @@
             case_data = pre2017.parse(case_soup, case_id)
         else:
             case_data = post2017.parse(case_soup, case_id)
-
+        #Adds county field to data
```

Comment:
This will work! 

I think from a code architecture perspective, it would be cleaner to do something like 

```python
        if odyssey_version < 2017:
           case_data = pre2017.parse(case_soup, case_id, args.county)
        else:
           case_data = post2017.parse(case_soup, case_id, args.county)
```
and then add the argument within the pre2017 / post2017 classes (or even have a class for each county). All the other logic for adding args to the case_data dict is contained within those methods, so I could see someone reading that file later on and wondering where "county" gets added in. Plus, there will probably be some other county-specific parsing logic that will need to be contained within those classes.

Reply:
This makes sense! I'll move this to the pre2017/post2017 classes.